.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,13 +1,31 @@
 ---
-name: Bug report
+name: "\U0001F41B Bug report"
 about: Create a bug report to help us improve
+labels: type:bug
 
 ---
 
-**What is the current behavior? Describe the bug**
+## 📝 Provide detailed reproduction steps (if any)
```

Comment:
could be good to mention our fiddle here? if someone doesn't know it it could be very helpful

and maybe also suggest to use screenshot and video if it makes the bug more clear, I think some people could not think about it and could be very helpful for us